app/models/concerns/api/v2/interfaces_controller_extensions.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V2
+      module InterfacesControllerExtensions
+        extend Apipie::DSL::Concern
+
+        update_api(:create, :update) do
+          param :interface, Hash do
+            param :execution, :bool, :desc => N_('Is the execution interface used for remote execution')
```

Comment:
The other flags have the description in the format of "Should this interface be used for $SOMETHING?", it would be nice to stick to it

Reply:
right, changed.